x-pack/plugins/monitoring/server/lib/elasticsearch/nodes/get_nodes/map_nodes_info.js

```diff
@@ -35,7 +35,7 @@ export function mapNodesInfo(nodeHits, clusterStats, shardStats) {
         isOnline,
         nodeTypeLabel: nodeTypeLabel,
         nodeTypeClass: nodeTypeClass,
-        shardCount: get(shardStats, `nodes[${sourceNode.uuid}].shardCount`, null),
+        shardCount: get(shardStats, `nodes[${sourceNode.uuid}].shardCount`, 0),
```

Comment:
👍 It looks good to fix the bug this way. It will help the UI make sense of the data and be able to sort it correctly when the table is sorted by number of shards.